app/models/conference.rb

```diff
@@ -1,6 +1,5 @@
 ##
 # This class represents a conference
-# rubocop:disable Style/ClassLength
 class Conference < ActiveRecord::Base
```

Comment:
I would want to try to make sure we're not making this larger. Or if we have to, increase the limit in the config file.